wsgi.py

```diff
@@ -26,9 +27,10 @@ def load(self):
 
 # We use the port 5000 as default, otherwise we set the HTTP_PORT env variable within the container.
 if __name__ == '__main__':
+    HTTP_PORT = str(os.environ.get('HTTP_PORT', "5000"))
```

Comment:
Why removing this from settings ? The old code made more sense and was cleaner, concentrating all settings in the settings module.